hail/python/hailtop/hailctl/batch/billing/list_billing_projects.py

```diff
@@ -1,15 +1,13 @@
 import sys
-import tabulate
-from ..batch_cli_utils import make_formatter
+from ..batch_cli_utils import make_formatter, choices
 
 
 def init_parser(parser):
     parser.add_argument('-o', type=str, default='grid',
-                        help='specify output format (json, yaml, csv, tsv, or any tabulate format)')
+                        choices=choices)
 
 
 def main(args, pass_through_args, client):  # pylint: disable=unused-argument
-    choices = ['json', 'yaml', *tabulate.tabulate_formats]
     if args.o not in choices:
         print('invalid output format:', args.o, file=sys.stderr)
         print('must be one of:', *choices, file=sys.stderr)
```

Comment:
These three lines can now be removed as args.o is guaranteed to be one of those options at this point.

Reply:
I don't think args.o is guaranteed to be one of those options when those lines of code execute. I added an assertion in the make_formatter function in batch_cli_utils.py, but that runs after these lines

Reply:
Try running `hailctl batch list -o asdf`. I get an error claiming that i've used an invalid choice, but I don't see these print statements. That makes me think that `argparse` is outputting the error and exiting, making these lines dead code.